ccd-definition/CaseField.json

```diff
@@ -809,6 +809,13 @@
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "respondStatementOfTruthLabel",
```

Comment:
You will need to add Authorisation privilages for this field to determine what user can see this field on the event!

Reply:
Added Authorisation. please check